README.md

```diff
@@ -1,2 +1,242 @@
-# learn-user-testing
-Learn how to test your app with real humans 
+# Learn User Experience Testing
+Learn how to test your app with real humans
+
+## Why?
+This is _probably_ the most important/useful skill any "_creative technologist_"
+has. 💪
+
+Well conducted user experience testing helps you ensure that what you are
+planning to build is going to solve the problem for your users that you set out
+to solve. ✅
+
+It prevents you from wasting time ⏳/ effort 😓/ money 💰on guessing 🤔❓what your
```

Comment:
While it would be _nice_ if UX testing would "_prevent_" wasting time/effort, sadly, _often_ it does _not_.
Unless the UX designer has the appropriate incentive(s) and makes an _effort_ to construct their experiment(s) to _avoid_ asking **_leading_ questions**, UX testing _can_ lead to _invalid_ conclusions with a heavy ["***confirmation bias***"](https://en.wikipedia.org/wiki/Confirmation_bias).

Reply:
Starting with...
> Done properly, it prevents

Adding in 'what':
> User experience testing cannot be performed as a 'token gesture' to produce 
insightful results. Care must be taken to ensure that guidelines are followed 
properly so that the results are not invalidated by ["_**confirmation bias**_"](https://en.wikipedia.org/wiki/Confirmation_bias).

Editing lower down to include 'leading questions':
> Avoid using leading questions or all yes/no questions. Open ended questions 
encourage people to give more detail and depth of analysis.